--- LibXSLT.xs.orig	2008-11-05 14:19:41.000000000 +0100
+++ LibXSLT.xs	2009-09-07 22:40:21.000000000 +0200
@@ -347,7 +347,6 @@
 	    if (container == NULL) {
               croak("LibXSLT: perl-dispatcher: cannot create container RVT\n");
             }
-	    xsltRegisterLocalRVT(tctxt,container);
             ret = xmlXPathNewNodeSet(NULL);
             ret->boolval = 0;
             array_result = (AV*)SvRV(perl_result);
@@ -387,7 +386,6 @@
 	      if (container == NULL) {
 		croak("LibXSLT: perl-dispatcher: cannot create container RVT\n");
 	      }
-	      xsltRegisterLocalRVT(tctxt,container);
 	      tmp_node = xmlDocCopyNode(tmp_node1, container, 1);
 	      xmlAddChild((xmlNodePtr)container,tmp_node);
 	      xmlXPathNodeSetAdd(ret->nodesetval,tmp_node);
