--- Makefile.PL.orig	2009-09-07 22:38:23.000000000 +0200
+++ Makefile.PL	2009-09-07 22:38:38.000000000 +0200
@@ -47,7 +47,7 @@
             my $ver = backtick("$xsltcfg --version");
             my ($major, $minor, $point) = $ver =~ /(\d+)\.(\d+)\.(\d+)/g;
             die "VERSION" unless $major > 1 || $minor > 1 ||
-	      ($major>=1 and $minor >=1 and $point >= 18);
+	      ($major>=1 and $minor >=1 and $point >= 12);
             $config{LIBS} ||= backtick("$xsltcfg --libs");
             $config{INC} ||= backtick("$xsltcfg --cflags");
             print "ok\n";
@@ -55,7 +55,7 @@
         if ($@) {
             print "failed\n";
             if ($@ =~ /^VERSION/) { 
-                print STDERR "XML::LibXSLT needs libxslt version 1.1.18 or higher\n";
+                print STDERR "XML::LibXSLT needs libxslt version 1.1.12 or higher\n";
 		exit 0; # 0 recommended by http://cpantest.grango.org (Notes for CPAN Authors)
             }
             warn "*** ", $@ if $DEBUG;
